index.html

```diff
@@ -225,5 +225,27 @@ <h3>Skills</h3>
             </ul>
             </div>
         </section>
+        <section class="contact">
+            <div>
+            <h2>I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/{https://formspree.io/f/xeqdywdz}" method="post">
+                <input type="text" placeholder="Full name" maxlength="30">
+                <input type="text" placeholder="Email address" name="email">
+                <input class="inputfield" type="text" placeholder="Enter text here" maxlength="500">
+            </form>
+            <button>Get in touch</button>
```

Comment:
- [ ] This button type should take submit so that it would be connected to the form action. However, it takes submit by default but I think the closing curly bracket in the action attribute of the form is the hindering factor here. Kindly remove it.